playbooks/rfcs.md

```diff
@@ -39,6 +39,24 @@ Find the right repo:
 - Does it affect the whole dev team? (use [README](https://github.com/artsy/README))
 - Can't figure it out or needs to be private? (use [Potential][] as a fallback)
 
+Create a plan for how the RFC will be implemented. If the change is complex or large in scope consider defining a
+feedback process to iterate on the proposal as you implement and creating milestones and/or metrics to track
+progress.
+
+Decide who needs to know about this proposal, what role teammates will play in the process and how the RFC will be
+considered approved.
+
+- What is the scope of the proposal? Does the whole engineering team need to know or just a smaller group of
+  engineers?
+- Who should actively approve of this RFC? Who should give feedback?
+- Does the change make sense without further discussion from engineering leadership?
+
+Here are some examples of how an RFC could be resolved:
+
+- X% of the engineering team actively approves this change.
+- The point people of the impacted repos approve this change.
+- Engineering leadership approves the changes after collecting feedback from the team.
```

Comment:
I feel like this list is pointing towards the larger and thornier problem of how large-scale change happens within the engineering org. If I'm writing a major RFC, how exactly do I choose which of your example resolutions is the right one? If I choose the first one, what should the value of X be? Who actually are the point people for the repos? What are the circumstances in which leadership needs to be involved? Who even is "leadership", in this context?

Reply:
This issue is at the heart of this RFC. Right now there's no mechanism or guidelines for choosing any of this and it's up to the RFC creator to find the right way to build support for their RFC. In practice that means having a nuanced understanding of our org and knowing how support gets built and what the "right" kind of approval is for a given situation. It's largely tribal knowledge. So these suggestions are necessarily fuzzy because right now creating large change is ambiguous. I'm not able to answer your questions, and I don't necessarily even think that we should use these suggestions precisely because they're unclear. This RFC just takes a small step towards reducing ambiguity, though, by suggesting that we explicitly call out what the approval process is and debate it if necessary. But I'd love to see better-defined approval mechanisms or stronger guidelines for what to do when writing a major RFC!